module/usb_disk_mode.c

```diff
@@ -107,94 +122,13 @@ void tele_usb_disk() {
                 continue;
             }
 
```

Comment:
the following code is moved mostly verbatim to scene_serialization.c, just with calls to `file_write_buf`, etc. replaced with calls to the serializer interfaces.